src/main/webapp/js/receipt-analysis.js

```diff
@@ -12,6 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Loads the page if the user is logged in. Otherwise, redirects to the
+ * login page.
+ */
+function load() {
+  /* global loadPage */
+  loadPage(loadReceiptAnalysis);
```

Comment:
I dislike that it's not clear where the method loadPage() is from. I wonder if we can add an import or a comment?

Reply:
Added a comment. I tried importing, but that would require configuring all the js files as modules in the linter.